### PR TITLE
Meteor shower

### PR DESCRIPTION
**Pull request by haniacro**

### CODE REVIEW

File: 1_space_odyssey/js/Hani-9000/app.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    "\u{1F327} METEOR SHOWER::::::"
+  );
+  
+  var responses = [
+    "Call basecamp and ask for assistance, but they are far away",
+    "Do nothing, just push through",
+    "initiate core thrusters, punch 180degrees into dash-nav and warp away from danger",
+    "ask humanoid-9000 to fly out there and shield the ship from the debris, this action could harm the humanoid"
+  ];
+  var response = responses.Length;
```
Comment (mannuelf) [inline 1_space_odyssey/js/Hani-9000/app.js:135]: nice job yeah man.
spelling error: `esponses.Length;`
should be: `var response = responses.length;`

File: 1_space_odyssey/css/styles.css [hunk 2]
```diff
@@ -1,17 +1,34 @@
...
+  transition: fade ease-in;
 }
 
 h1 {
-  font-family: 'Ubuntu', sans-serif;
-  color: #FFF;
-  text-align: center;
-  margin: 50vh auto;
-  font-size: 3em;
+font-family: 'Ubuntu', sans-serif;
```
Comment (mannuelf) [inline 1_space_odyssey/css/styles.css:13]: please revert the CSS changes, they are not needed.

File: 1_space_odyssey/js/Hani-9000/app.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+var canWalk = true;
+var canRun = true;
+var canFly = false;
+var hasWeapon = false;
+var isInGoodMood = false;
+
+console.log(".-.-. Booting .-.-.");
+console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, hands, ears, canWalk, canFly, hasWeapon);
+
+if(hasJetPack === true) {
```
Comment (mannuelf) [inline 1_space_odyssey/js/Hani-9000/app.js]: 👍 
clean code, please indent the code nicely. notice the spaces and indents 
```js
if (condition) {
  ...
} else {
  ...
}
```

File: 1_space_odyssey/js/Hani-9000/app.js [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+  
+  // a.
+  var enemyCraft = true;
+  var photonLaser = true;
+  var photonLaserEnabled = true;
+  var bullets = 100;
+
+  if( enemyCraft && photonLaser && photonLaserEnabled){
+    if ( bullets <= 100) {
+   for(var i = 0; i < 100; i ++) {
```
Comment (mannuelf) [inline 1_space_odyssey/js/Hani-9000/app.js]: great, this is correct.

clean code, please indent this if block nicely .
scan this document: https://github.com/mannuelf/clean-code-javascript notice how the if blocks are indented. please follow that as code style and stick with it.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,5 @@
+<<<<<<< HEAD
```
Comment (mannuelf) [inline README.md]: please revert this file to its original state and git push.

File: LICENSE [hunk 6]
```diff
@@ -1,21 +1,21 @@
-MIT License
```
Comment (mannuelf) [inline LICENSE:1]: please revert this file to its original state and git push.

File: 1_space_odyssey/js/mannuel-9000/sander-9000/sander-app.js [hunk 7]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD:1_space_odyssey/js/mannuel-9000/sander-9000/sander-app.js
```
Comment (mannuelf) [inline 1_space_odyssey/js/mannuel-9000/sander-9000/sander-app.js]: please revert this file to its original state and git push.

File: 1_space_odyssey/js/SAM-9000/app.js [hunk 8]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD:1_space_odyssey/js/SAM-9000/app.js
```
Comment (mannuelf) [inline 1_space_odyssey/js/SAM-9000/app.js]: please revert this file to its original state and git push.

**Review by mannuelf — CHANGES_REQUESTED** [review]

File: 1_space_odyssey/css/styles.css (hunk 2 above)
Comment (haniacro) [reply 1_space_odyssey/css/styles.css:13]: I didn't touch the CSS file, I only deleted the merge conflict issues

**Pull request closed without merging**